openmrs/apps/reports/GEORGETOWN_HIV_PATIENT_ON_TREATMENT_REPORT/sql/georgetownHIVPatientOnTreatmentReport.sql

```diff
@@ -16,7 +16,7 @@ SELECT
     getActiveARVWithLowestDispensationPeriod(pat.patient_id, "2000-01-01", "2100-01-01") as "currentRegimen",
     getPatientMostRecentProgramAttributeCodedValue(pat.patient_id, "397b7bc7-13ca-4e4e-abc3-bf854904dce3", "en") as "currentLine",
     IF(patientIsEligibleForVL(pat.patient_id), "Yes", "No") as "eligibilityForVl",
-    getDateMostRecentARVAppointmentWhereStatusMissed(pat.patient_id, "#endDate#") as "lastAppointmentDate",
+    getARTAppointmentOnOrAfterDate(pat.patient_id, GREATEST("#startDate#", getLastArvPickupDate(pat.patient_id, "2000-01-01", "#endDate#"))) as "lastAppointmentDate",
```

Comment:
@tmvumbi2 this specific use case was suppose to take the lasAppointmentDate(a date in the future) but at the moment it's not doing that